app/scripts/directives/oscPersistentVolumeClaim.js

```diff
@@ -0,0 +1,71 @@
+"use strict";
+
+angular.module("openshiftConsole")
+  .directive("oscPersistentVolumeClaim", function($filter){
+    return {
+      restrict: 'E',
+      scope: {
+        claim: "=model",
+        showNameInput: "=",
+        showAmountInput:"="
```

Comment:
Would we ever hide this? I think I'd just simplify the directive and just take in "claim" and nothing else until we need it.